### PR TITLE
MH3 10 double faced cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/drowner_of_truth_drowned_jungle.txt [hunk 1]
```diff
@@ -0,0 +1,18 @@
+Name:Drowner of Truth
+ManaCost:5 GU GU
+Types:Creature Eldrazi
+PT:7/6
+K:Devoid
+T:Mode$ SpellCast | ValidCard$ Card.Self | ConditionManaSpent$ C | Execute$ TrigToken | TriggerDescription$ When you cast this spell, if {C} was spent to cast it, create two 0/1 colorless Eldrazi Spawn creature tokens with "Sacrifice this creature: Add {C}."
```
Comment (Simisays) [inline forge-gui/res/cardsfolder/upcoming/drowner_of_truth_drowned_jungle.txt:6]: ConditionManaSpent$ C  is not the correct param, any ideas ?

File: forge-gui/res/cardsfolder/upcoming/disciple_of_freyalise_garden_of_freyalise.txt [hunk 2]
```diff
@@ -0,0 +1,23 @@
+Name:Disciple of Freyalise
+ManaCost:3 G G G
+Types:Creature Elf Druid
+PT:3/3
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigSac | TriggerDescription$When CARDNAME enters the battlefield, you may sacrifice another creature. If you do, you gain X life and draw X cards, where X is that creature's power.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/disciple_of_freyalise_garden_of_freyalise.txt:5]: spacing

File: forge-gui/res/cardsfolder/upcoming/disciple_of_freyalise_garden_of_freyalise.txt [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+SVar:TrigSac:DB$ Sacrifice | Amount$ 1 | SacValid$ Creature.Other | RememberSacrificed$ True | SubAbility$ GainLife
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/disciple_of_freyalise_garden_of_freyalise.txt:6]: just write it as cost

File: forge-gui/res/cardsfolder/upcoming/drowner_of_truth_drowned_jungle.txt (hunk 1 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/drowner_of_truth_drowned_jungle.txt:6]: try without "Condition"

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/drowner_of_truth_drowned_jungle.txt]: No succes

Comment (squee1968) [reply forge-gui/res/cardsfolder/upcoming/drowner_of_truth_drowned_jungle.txt]: Check out Desecrate Reality

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/drowner_of_truth_drowned_jungle.txt]: `CheckSVar$ CastSA>Count$Adamant_1.Colorless.1.0`

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/drowner_of_truth_drowned_jungle.txt]: Thank you, working now

**Pull request merged**